.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+- React-Native tools version:
```

Comment:
Leave it at "React-Native version". We are interested in react-native's version and NOT react-native-cli's version?